test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +54,33 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+TEST(ProcSysNetIpv4IpForward, Exists) {
+  auto fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/proc/sys/net/ipv4/ip_forward", O_RDWR));
+
+  fd.reset();
+}
+
+TEST(ProcSysNetIpv4IpForward, CanReadAndWrite) {
+  auto const fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/proc/sys/net/ipv4/ip_forward", O_RDWR));
+
+  char buf = 101;
+  EXPECT_THAT(PreadFd(fd.get(), &buf, sizeof(buf), 0),
+              SyscallSucceedsWithValue(sizeof(buf)));
+
+  EXPECT_TRUE(buf == '0') << "unexpected ip_forward: " << buf;
+
+  char to_write = '1';
```

Comment:
Can you make this a constant?

Example:
`constexpr char kToWrite = '1';`

Reply:
added constexpr